Makefile

```diff
@@ -14,10 +14,12 @@ $(TEMPDIR):
 
 .PHONY: bootstrap-tools
 bootstrap-tools: $(TEMPDIR)
-	VERSION=v1.24.0 TMPDIR=.tmp bash .github/scripts/goreleaser-install.sh
-	curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b .tmp v1.0.1
-	curl -sSfL https://github.com/sigstore/cosign/releases/download/v2.2.3/cosign-linux-amd64 -o .tmp/cosign
-	chmod +x .tmp/goreleaser .tmp/cosign .tmp/syft
+	command -v .tmp/goreleaser || VERSION=v1.24.0 TMPDIR=.tmp bash .github/scripts/goreleaser-install.sh
```

Comment:
I know you just inherited this but I think the relative directory `hack/bin/` or `hack/tools/bin` is more idiomatic than `.tmp/` for installing and running supporting binaries at runtime. Maybe a follow-up?

Reply:
I would love to do it either in a follow up or here, whatever merges first :)

Reply:
done in this MR.